### ISSUE TITLE
Linear Shell Analysis

### ISSUE THREAD

**Issue by msandre** [KratosMultiphysics/GiDInterface#506]
Hi,
we are testing v6.0.0 for the upcoming semester and noticed that for structural analysis, one can only select shell element if it is a nonlinear analysis. Is it possible to also make linear shell analysis available in the next release?

**Comment by RiccardoRossi** [comment]
@jginternational

**Comment by philbucher** [comment]
Yes, I implemented all elements in structuralMechanics nicely available in the Interface more than half a year ago

**Comment by msandre** [comment]
Ok thanks :+1:
